src/requireHook.ts

```diff
@@ -4,14 +4,23 @@ import mysqlHook from "./hooks/mysql";
 import pgHook from "./hooks/pg";
 import prismaHook from "./hooks/prisma";
 import sqliteHook from "./hooks/sqlite";
+import standardLibsHook from "./hooks/standardLibs";
 
 interface Hook {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   (mod: any, id?: string): any;
```

Comment:
You can probably change the signature to require `id` and avoid the assert in *standardLibsHook*. I can't remember why it's here but it seems it will always be present.